### PR TITLE
Make sure tokens balance can be fetched

### PR DESCRIPTION
**Pull request by rasom**
needed for https://github.com/status-im/status-react/issues/11674

We need to set to wallet `Ethclient` before we check history the first time as it might not happen on mobile network.

### CODE REVIEW

File: api/geth_backend.go [hunk 1]
```diff
@@ -628,6 +628,11 @@ func (b *GethStatusBackend) startNode(config *params.NodeConfig) (err error) {
 	}); err != nil {
 		return
 	}
+	wallet, err := b.statusNode.WalletService()
+	if err != nil {
+		return err
+	}
+	wallet.SetClient(b.statusNode.RPCClient().Ethclient())
```
Comment (Samyoul) [inline api/geth_backend.go]: The var `wallet` collides with the package `github.com/status-im/status-go/services/wallet`, although this code is compliable it makes any reference to "wallet" in this function ambiguous. An easy fix is to rename the var to `walletService` or `w` or something cooler.

**Review by Samyoul — APPROVED** [review]
Looks good, just a var / package name collision and we need a VERSION bump 😄

File: api/geth_backend.go (hunk 1 above)
Comment (rasom) [reply api/geth_backend.go]: fixed

File: services/wallet/service.go [hunk 2]
```diff
@@ -51,6 +51,11 @@ func (s *Service) GetFeed() *event.Feed {
 	return s.feed
 }
 
+// SetClient sets ethclient.
+func (s *Service) SetClient (client *ethclient.Client) {
```
Comment (Samyoul) [inline services/wallet/service.go]:
```suggestion
func (s *Service) SetClient(client *ethclient.Client) {
```

**Review by Samyoul — COMMENTED** [review]
Try this

**Pull request merged**